x/pairing/keeper/scores/stake_req.go

```diff
@@ -17,7 +17,14 @@ func (sr *StakeReq) Init(policy planstypes.Policy) bool {
 
 // Score calculates the the provider score as the normalized stake
 func (sr *StakeReq) Score(score PairingScore) math.Uint {
-	return sdk.NewUint(score.Provider.Stake.Amount.Uint64())
+	stake_score := score.Provider.Stake.Amount
+	if score.Provider.DelegateLimit.Amount.LT(score.Provider.DelegateTotal.Amount) {
```

Comment:
this should be a helper in dualstaking (or - as suggested in CNS-531 review - possibly stored on the stakeEntry)

Reply:
done 0aa5c663504a1f29e262ff8666deee1af71ce302